app/views/shoping_lists/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>General shoping list</h1>
+<table>
+  <tr><th>Food</th><th>quantity</th><th>price</th></tr>
+  <% @shoping_lists.each do |food| %>
+  <tr>
+    <td><%= food.name%></td>
+    <td><%= food.quantity%></td>
+    <td>$<%= food.price%></td>
+  </tr>
+  <% end %>
+
+</table>
+    <div class="action-buttons">
+    <%= button_to 'Home', user_path(current_user), method: :get, class: "success"%>
+    </div>
+
```

Comment:
- [ ] As required for the project, kindly make sure you also add the `General shopping list view` with their expected features as mentioned in the [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md).
![Screenshot from 2022-09-02 03-16-20](https://user-images.githubusercontent.com/83042742/188038505-349236f5-d912-420c-a054-ca7f8b7a8533.png)

Reply:
Fix and merged with [6d855be](https://github.com/Tadesse-Alemayehu/Recipe-app/pull/33/commits/6d855bec0adc306aa615750ba7a31df15fae63ce) commit hash.